conf/v2_route/user.api

```diff
@@ -321,6 +321,30 @@ DELETE   /user/:userId/unsave/:challengeId          @org.maproulette.framework.c
 GET     /user/:userId/savedTasks                    @org.maproulette.framework.controller.UserController.getSavedTasks(userId:Long, challengeIds:String ?= "", limit:Int ?= 10, page:Int ?= 0)
 ###
 # tags: [ User ]
+# summary: Retrieves Users Locked Tasks
+# description: Retrieves a list of all the tasks the user with the matching id has locked
+# responses:
+#   '200':
+#     description: The retrieved Tasks
+#     content:
+#       application/json:
+#         schema:
+#           type: array
+#           items:
+#             $ref: '#/components/schemas/org.maproulette.framework.model.Task'
+#   '401':
```

Comment:
Nit: where's 404?

Reply:
>      description: If User or Task for provided ID's is not found.

The wording is wrong, there's no relation to the requesting "User" and a 404. It's only the userId. So keep it to "description: userId is not known" or something like that